EIPS/eip-663.md

```diff
@@ -72,14 +89,14 @@ This has no effect on backwards compatibility because the opcodes were not previ
 Given variable `n`, which equals to `imm + 1`, for `1 <= n <= 256`:
 
     - `DUPN imm` to fail validation if `stack_height < n`.
-    - `SWAPN imm` to fail validation if `stack_height < (n + 1)`.
+    - `EXCHANGE imm` to fail validation if `stack_height < n + m`.
```

Comment:
I think my other comment got lost: `SWAPN` cases got lost in merges, this entire section needs a revisit at the end. Also specifically it's `n + m + 1` now for EXCHANGE